model/github_app_auth_test.go

```diff
@@ -0,0 +1,61 @@
+package model
+
+import (
+	"testing"
+
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestFindOneGithubAppAuth(t *testing.T) {
+	assert := assert.New(t)
+
+	require.NoError(t, db.Clear(GitHubAppAuthCollection),
+		"Error clearing collection")
+
+	key := []byte("I'm private!")
+	githubAppAuth := GithubAppAuth{
+		Id:         "mongodb",
+		AppId:      1234,
+		PrivateKey: key,
+	}
+	change, err := githubAppAuth.Upsert()
+	assert.NotNil(change)
```

Comment:
I think we should do require.NoError(t, err) and then assert.NotNil(change) just to prevent any nil pointer exceptions (and same for the below where they return an error